app/src/main/kotlin/org/vinaygopinath/launchchat/screens/main/MainActivity.kt

```diff
@@ -87,7 +92,14 @@ class MainActivity : AppCompatActivity() {
         if (phoneNumbers.isEmpty()) {
             phoneNumberInputLayout.error = getString(R.string.toast_invalid_phone_number)
         } else if (phoneNumbers.size != 1) {
-            // TODO Multiple phone numbers detected
+            showPhoneNumberSelectionDialog(phoneNumbers) { selectedNumber ->
+                val message = messageInput.text.toString().trim()
+                try {
+                    startActivity(lambda(selectedNumber, message))
+                } catch (e: ActivityNotFoundException) {
+                    showToast(errorToast)
+                }
```

Comment:
DRY: This is very similar to the code that exists just above these lines. Thoughts on refactoring to a private method?